packages/core/src/stateUtils.ts

```diff
@@ -1582,7 +1582,7 @@ export function resolveMicroTransition<
     return inertState as any;
   }
 
-  const children = !currentState._initial ? { ...currentState.children } : {};
```

Comment:
q: why this was removed? won't it result in mutability bugs?

Reply:
f0de1acffe0c0471ebac622fdcebaaa2eb770e92